youtube_dl/extractor/rumble.py

```diff
@@ -26,10 +28,18 @@ class RumbleEmbedIE(InfoExtractor):
     }, {
         'url': 'https://rumble.com/embed/ufe9n.v5pv5f',
         'only_matching': True,
+    }, {
+        'url': 'https://rumble.com/vhlrar-mike-lindell-to-confront-brian-kemp-and-doug-ducey-over-election-fraud.html',
+        'only_matching': True,
     }]
 
     def _real_extract(self, url):
-        video_id = self._match_id(url)
+        if re.match(r'https?://(?:www\.)?rumble\.com/(.*?)\.html', url):
+            video_id = ""
+            content, urlh = self._download_webpage_handle(url, video_id)
+            video_id = re.findall(r'"embedUrl":"https://rumble\.com/embed/(.*?)/"', content)[0]
```

Comment:
Might the site start sending white space around the `:`, which would be matched by `\s*:\s*` in the regex?